Content.Server/GameObjects/EntitySystems/RoundEndSystem.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Threading;
+using Content.Server.Interfaces.GameTicking;
+using Robust.Shared.GameObjects.Systems;
+using Robust.Shared.IoC;
+using Timer = Robust.Shared.Timers.Timer;
+
+namespace Content.Server.GameObjects.EntitySystems
+{
+    public class RoundEndSystem : EntitySystem
+    {
+#pragma warning disable 649
+        [Dependency] private IGameTicker _gameTicker;
+#pragma warning restore 649
+
+        private CancellationTokenSource _roundEndCancellationTokenSource = new CancellationTokenSource();
+        public bool IsRoundEndCountdownStarted { get; private set; }
+        public int RoundEndCountdownTime { get; set; } = 5000;
```

Comment:
Store this as a `TimeSpan` or something.

Reply:
Also isn't 5 seconds kinda short?

Reply:
Yeah, it was short on purpose for debugging purposes but I changed it to 4 minutes now. Also changed this to use TimeSpan.